lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -927,88 +963,22 @@ def test_cache_map_traverse_cmap(
+            c_map = tb.find_entry(Path(file_path))
+            if file_type is CacheType.DIRECTORY:
+                assert sorted(c_map.keys()) == [
+                    "children",
+                    "details",
+                ], "Directory should have children and details"
+            else:
+                assert sorted(c_map.keys()) == [
+                    "details"
+                ], "Non-directory should have only details"
             assert c_map["details"].location == Path(location)
             assert c_map["details"].name == name
             assert c_map["details"].resolve_path == resolve_path
-            assert c_map["details"].resolve_type == resolve_type
+            assert c_map["details"].resolve_type is resolve_type
             assert c_map["details"].size == size
-            assert c_map["details"].type == file_type
-
-    @pytest.mark.parametrize(
-        "file_path, expected_msg",
-        [
-            (
-                "dir_name/subdir1/f11.txt",
-                {
-                    "location": Path("dir_name/subdir1/f11.txt"),
-                    "name": "f11.txt",
-                    "resolve_path": None,
-                    "resolve_type": None,
-                    "size": 14,
-                    "type": CacheType.FILE,
-                },
-            ),
-            (
-                "dir_name/subdir1",
-                {
-                    "directories": ["subdir11", "subdir12", "subdir13", "subdir14"],
-                    "files": ["f11.txt"],
-                    "location": Path("dir_name/subdir1"),
-                    "name": "subdir1",
-                    "resolve_path": None,
-                    "resolve_type": None,
-                    "size": None,
-                    "type": CacheType.DIRECTORY,
-                },
-            ),
-            (
-                "dir_name/subdir1/subdir11",
-                {
-                    "directories": [],
-                    "files": [],
-                    "location": Path("dir_name/subdir1/subdir11"),
-                    "name": "subdir11",
-                    "resolve_path": None,
-                    "resolve_type": None,
-                    "size": None,
-                    "type": CacheType.DIRECTORY,
-                },
-            ),
-            (
-                "dir_name/subdir1/subdir14/subdir141/f1413_sym",
-                {
-                    "location": Path("dir_name/subdir1/subdir14/subdir141/f1413_sym"),
-                    "name": "f1413_sym",
-                    "resolve_path": Path("dir_name/subdir1/subdir14/subdir141"),
-                    "resolve_type": CacheType.DIRECTORY,
-                    "size": None,
-                    "type": CacheType.SYMLINK,
-                },
-            ),
-        ],
-    )
-    def test_cache_map_get_info_cmap(
-        self, make_logger, monkeypatch, tmp_path, file_path, expected_msg
-    ):
```

Comment:
Why did you remove this test?  I see that you've added a similar test in `test_datasets_contents.py`, but that's for a different "unit".